blackbox/docs/appendices/upgrade-guides/3.0.0-upgrade-guide.rst

```diff
@@ -0,0 +1,98 @@
+.. _3.0.0_upgrade_guide:
+
+=======================
+Crate 3.0 Upgrade Guide
+=======================
+
+Upgrade Strategy
+================
+
+Upgrading to 3.0.0 is only possible from a 2.X.Y release. It is advisable to to
```

Comment:
Should read: `3.0.x` and `2.x`

Reply:
Please check all occurrences. Writing `3.0.x` will make keep the upgrade guide valid when there is a new 3.0 hotfix.